app/views/data_access/retrieval.html.erb

```diff
@@ -24,26 +24,23 @@
       <p>
         Click the "View/Download Data Links" button to view or download a file containing links to your data.
         You may bookmark this page for later access.
-        A browser download manager plugin such as Firefox's
-        <a class="link" href="https://addons.mozilla.org/en-US/firefox/addon/downthemall/">DownThemAll!</a>
-        can assist you in managing a large number of download links.
       </p>
-      <ul class="data-access-collections-list" data-bind="foreach: ui.projectList.collectionsToDownload">
-        <li class="data-access-collection">
-          <p class="data-access-collection-name" data-bind="text: title"></p>
+      <ul class="data-access-collections-list data-access-collections-list-panel" data-bind="foreach: ui.projectList.collectionsToDownload">
+        <li class="data-access-collection complete">
+          <h3 class="data-access-collection-name" data-bind="text: title"></h3>
           <div class="data-access-collection-actions">
             <!-- ko foreach: links -->
-            <a class="button text-button" href="#" target="_blank" data-bind="text: title, attr: {href: url, title: tooltip}" data-tooltip="true" data-placement="bottom"></a>
+            <a class="button button-small text-button" href="#" target="_blank" data-bind="text: title, attr: {href: url, title: tooltip}" data-tooltip="true" data-placement="bottom"></a>
             <!-- /ko -->
           </div>
         </li>
       </ul>
     </section>
     <section class="data-access-content data-access-collection-only" data-bind="visible: ui.projectList.collectionOnly().length > 0">
```

Comment:
@trevorlang you can kill this entire section. There is no way to download collection-only data anymore. I just killed this section and ran tests locally, nothing broke. @abbottry maybe you can take a quick look to see if there's anything else associated with this section that can also be removed?

Reply:
👍Will do